pkg/services/accesscontrol/resourcepermissions/api.go

```diff
@@ -186,6 +238,19 @@ func (a *api) setTeamPermission(c *contextmodel.ReqContext) response.Response {
 	return permissionSetResponse(cmd)
 }
 
+// swagger:route POST /access-control/:resource/:resourceID/builtInRoles/:builtInRole enterprise,access_control setResourcePermissionsForBuiltInRole
+//
+// Set resource permissions for a built-in role.
+//
+// Assigns permissions for a resource by a given type (`:resource`) and `:resourceID` to a built-in role.
+// Allowed resources are `datasources`, `teams`, `dashboards`, `folders`, and `serviceaccounts`.
```

Comment:
nit: we could maybe also list allowed builtin roles.

Reply:
The problem is that the builtin roles are named different for some resources (e.g. datasources). I'm not sure of a simple way to handle that unless we display a table or list for each resource. That might be ok, I'll play around with it.

Reply:
Oh, that's weird, I didn't realise that 😢 No worries then, better no info than incorrect info.